pyinfra/modules/iptables.py

```diff
@@ -0,0 +1,41 @@
+# pyinfra
+# File: pyinfra/modules/iptables.py
+# Desc: manage iptables rules
+
+'''
+The iptables modules handles iptables rules
+'''
+
+from __future__ import unicode_literals
+
+from pyinfra.api import operation
+
+
+@operation
+def forward(state, host, dport, to_ip, to_port=None,
+            interface='eth0', protocol='tcp'):
```

Comment:
Can we add a `present=True` kwarg to make it also possible to remove rules that are undesired?